cou/cli.py

```diff
@@ -16,11 +16,17 @@
 """Entrypoint to the 'charmed openstack upgrader'."""
 import argparse
 import logging
+import logging.handlers
+import os
+import pathlib
 import sys
 from typing import Any
 
 from cou.steps.plan import apply_plan, dump_plan, generate_plan
 
+HOME = os.getenv("HOME")
+COU_DIR_LOG = os.getenv("COU_DIR_LOG", f"{HOME}/.local/share/cou/log")
```

Comment:
not-blocking: maybe we can make Path directly here, e.g. `pathlib.Path(os.getenv(...))`